bootstrap/environment/variables.go

```diff
@@ -456,3 +460,21 @@ func logEnvironmentOverride(lc logger.LoggingClient, name string, key string, va
 	}
 	lc.Infof("Variables override of '%s' by environment variable: %s=%s", name, key, valueStr)
 }
+
+// GetURIRequestTimeout gets the configuration request timeout value from an environment variable (if it exists)
+// or uses default value.
+func GetURIRequestTimeout(lc logger.LoggingClient) time.Duration {
+	envValue := os.Getenv(envKeyFileURITimeout)
+	if len(envValue) <= 0 {
+		return defaultFileURITimeout
```

Comment:
just a nit pick: all other cases having logging either warn or info. should we also add logging here?

Reply:
Would a warning similar to line 474 work?

Reply:
yes, if you want to log it

Reply:
The other is logging an Override of the default. We don't log all our config settings, only the Overrides.